trackintel/preprocessing/staypoints.py

```diff
@@ -17,6 +17,7 @@ def generate_locations(
     agg_level="user",
     print_progress=False,
     n_jobs=1,
+    activities_only=False,
```

Comment:
imo this parameter should appear right after `agg_level` as it affects the generation (and not like `n_jobs` that only control the performance)

Reply:
Yes I agree that would be nicer. 
But my thought process behind that was that it might breaks code that only uses positional arguments and not keyword arguments.
````python
generate_locations(sp, "dbscan", 100, 1, "haversine", "user", True)
````
would behave differently for example.
But I would guess that no code was written that way where that would apply :D

Reply:
I changed the order of the keyword arguments.